run-bench.sh

```diff
@@ -5,13 +5,33 @@ Usage: run-bench.sh [OPTIONS] SIMULATOR
 Run NSuite benchmarks for SIMULATOR.
 
 Options:
-    --prefix=PATH    Use PATH as base for working directories.
-    --model=MODEL    Run benchmark MODEL.
-    --config=CONFIG  Run benchmarks with configuration CONFIG.
-    SIMULATOR        One of: arbor, neuron, or coreneuron.
+    --prefix=PATH        Use PATH as base for working directories.
+    --model=MODEL        Run benchmark MODEL.
+    --config=CONFIG      Run benchmarks with configuration CONFIG.
+    -o, --output=FORMAT  Override default path to benchmark outputs.
```

Comment:
This is inconsistent with the `--`-only formatting of the other command line args.